cmsplugin_cascade/tests/test_button.py

```diff
@@ -34,4 +34,4 @@ def test_internal_link(self):
             'link': {'pk': page.id, 'model': 'cms.Page', 'type': 'cmspage'}, 'target': ''}
         model_instance = add_plugin(self.placeholder, BootstrapButtonPlugin, 'en', glossary=glossary)
         html = model_instance.render_plugin({})
-        self.assertHTMLEqual(html, '<a href="/"  class="btn btn-success">HOME</a>')
+        self.assertHTMLEqual(html, '<a href="/en/"  class="btn btn-success">HOME</a>')
```

Comment:
As seen from Travis log, the output of this tests depends on the version of djangoCMS involved as it passes on some and fails on others. The redirect URL can be either '/en/' or '/en-us'